wire-ios/Wire-iOS/Sources/UserInterface/Availability/TitleView.swift

```diff
@@ -24,7 +24,7 @@ class TitleView: UIView, DynamicTypeCapable {
     // MARK: - Properties
     var titleColor: UIColor?
     var titleFont: FontSpec?
-    var tapHandler: ((UIButton) -> Void)?
+    var tapHandler: (UIButton) -> Void = { _ in }
```

Comment:
What is the advantage of having this filled with empty implementation? For me it's usually clearer to see `tapHandler = nil` when debugging for example.

Reply:
I guess there's manly the disadvantage you mentioned. 
I just prefer to use a default value instead of a redundant optional. (e.g. empty string instead of `nil`, no-op closure instead of `nil` etc.)